packages/taraxa-ui/src/components/BarChart/BarChart.tsx

```diff
@@ -23,7 +23,7 @@ ChartJS.register(
   Legend
 );
 
-const DEFAULT_OPTIONS = {
+const useTick = (tick: string) => ({
```

Comment:
I would rename this because if you use the word 'use' it is considered as a React Hook. getTick or returnTick or something similar.